source/Engine.cpp

```diff
@@ -266,8 +266,6 @@ void Engine::Place()
 		if(!ship->IsParked() && ship->GetSystem())
 			ships.push_back(ship);
 	
-	// Add NPCs to the list of ships. Fighters have to be assigned to carriers,
-	// and all but "uninterested" ships should follow the player.
```

Comment:
Is this comment now no longer valid?

Reply:
Yes, since this function only directly deals with player's ships.